### PR TITLE
DDP-6329 add full project names for CMI studies

### PR DESCRIPTION
**Pull request by yufengwng**
## Context

This PR adds in the full names for the existing CMI studies. Also fixes a bug. These name changes have already been applied to `dev` env so you can check it out there (once bug is fixed).

## Checklist

- [x] I have labeled the type of changes involved using the `C-*` labels.
- [x] I have assessed potential risks and labeled using the `R-*` labels.
- [x] I have considered error handling and alerts, and added `L-*` labels as needed.
- [x] I have considered security and privacy, and added `I-*` labels as needed
- [x] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.

## FUD Score

- [x] :relaxed: All good, business as usual!

## Testing

- [x] I have written zero automated tests but have poked around locally to verify proper functionality

## Release

For deployed envs test/staging/prod, we'll need to run study-builder
```
$ ./render.sh v1 {env} {study}
$ java -Dconfig.file=... -jar target/StudyBuilder.jar ... {study} --run-task UpdateStudyDetails
```

### CODE REVIEW

File: pepper-apis/src/main/java/org/broadinstitute/ddp/service/WorkflowService.java [hunk 1]
```diff
@@ -172,7 +172,7 @@ private String getStudyName(Handle handle, String userGuid, String studyGuid) {
         if (preferredLanguageCode != null) {
             List<StudyI18nDto> studyTranslations = translationDao.findTranslationsByStudyId(study.getId());
             for (StudyI18nDto dto : studyTranslations) {
-                if (dto.getLanguageCode() != null && dto.getLanguageCode() == preferredLanguageCode) {
+                if (dto.getLanguageCode() != null && dto.getLanguageCode().equals(preferredLanguageCode)) {
```
Comment (yufengwng) [inline pepper-apis/src/main/java/org/broadinstitute/ddp/service/WorkflowService.java:175]: Found a bug: we should compare strings with `equals()`.

File: study-builder/studies/angio/study.conf [hunk 2]
```diff
@@ -9,7 +9,7 @@
         "guid": "cmi"
     },
     "study": {
-        "name": "Angiosarcoma Project",
+        "name": "ascproject",
```
Comment (yufengwng) [inline study-builder/studies/angio/study.conf:12]: Just restoring the names in the config files. Note that we DID NOT push this `name` change to production when the file was changed, so we're just revert the change here.

**Review by ssettipalli — APPROVED** [review]

File: pepper-apis/src/main/java/org/broadinstitute/ddp/service/WorkflowService.java (hunk 1 above)
Comment (bskinner) [reply pepper-apis/src/main/java/org/broadinstitute/ddp/service/WorkflowService.java:175]: Nice catch! For anyone wondering why- when dealing with objects, `==` is reference equality (do both sides point to the same location in memory), whereas `equals(...)` is object equality (do the objects have the same content).

File: study-builder/studies/brain/study.conf [hunk 3]
```diff
@@ -35,7 +35,13 @@
...
     },
 
-    "studyDetails": [],
+    "studyDetails": [
+        {
+            "language": "en",
+            "name": "Brain Tumor Project",
+            "summary": null
+        }
+    ],
```
Comment (bskinner) [inline study-builder/studies/brain/study.conf:44]: This is a handy section to have as a reference.

File: study-builder/studies/mbc/study.conf [hunk 4]
```diff
@@ -37,7 +37,18 @@
...
+    {
+      "language": "en",
+      "name": "Metastatic Breast Cancer Project",
+      "summary": null
+    },
+    {
+      "language": "es",
+      "name": "Metastatic Breast Cancer Project",
+      "summary": null
+    },
```
Comment (bskinner) [inline study-builder/studies/mbc/study.conf:50]: Do we need the `es` translation if they're both identical?

**Review by bskinner — APPROVED** [review]
Looks good! Just had a question about the `escproject` changes.

File: study-builder/studies/mbc/study.conf (hunk 4 above)
Comment (yufengwng) [reply study-builder/studies/mbc/study.conf:50]: I think so, because the code looks it up by language and if it doesn't find it, I don't think it will automatically assume to grab the English one.

Comment (bskinner) [reply study-builder/studies/mbc/study.conf:50]: Do you think this should be tracked in Jira? From my understanding, this differs from our usual translation behavior of falling back to the default language.

Comment (yufengwng) [reply study-builder/studies/mbc/study.conf:50]: Yeah, I agree! I think if we didn't find the "preferred language" one, we should fallback to the study's "default language" and try to look for "study details" for that language. And if that fails also, then we fallback to "study dto name". I made a comment about this in @ssettipalli's PR: https://github.com/broadinstitute/ddp-study-server/pull/872#discussion_r669087343. Sampath, did we create a ticket to track this?

Comment (yufengwng) [reply study-builder/studies/mbc/study.conf:50]: In any case, I think there shouldn't be any issues with the configurations we have in this PR. For Spanish, I recall that we also want to show the English study name, so we should be good-to-go.
